app/controllers/dfe_sign_in_controller.rb

```diff
@@ -25,6 +30,27 @@ def callback
 
 private
 
+  def send_support_sign_in_confirmation_email
+    return unless FeatureFlag.active?('support_sign_in_confirmation_email')
+
+    return if cookies.signed[:sign_in_confirmation] == @local_user.id
```

Comment:
Should we namespace this cookie as `support_sign_in_confirmation` for explicitness?